packages/ui/src/auth/Register.tsx

```diff
@@ -14,6 +14,12 @@ type Props = {
 
 const Register: FunctionComponent<Props> = (props: Props) => {
 
+  const queryParams = parse(props.location.search,{decode:true});
+  const email: string = queryParams.email
```

Comment:
This is do the user does have to enter the email. When I send the email I add it as a queryParam and this way it gets pre filled